src/components/structures/MatrixChat.js

```diff
@@ -221,7 +221,7 @@ export default createReactClass({
         return {serverConfig: props};
     },
 
-    componentWillMount: function() {
+    componentDidMount: function() {
         SdkConfig.put(this.props.config);
```

Comment:
for consumers other than riot-web which already set the SdkConfig in advance this will need to happen pre-mount.